modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -37,8 +64,58 @@ The Deploy Application page opens:
 +
 image::deploying-to-cloudhub-8524f.png[]
 
+=== Deploy from CloudHub CLI
+
+To deploy a Mule app from the CloudHub Command Line Utility (CLI):
+
+. Verify that https://www.mulesoft.com/platform/saas/cloudhub-ipaas-cloud-based-integration[CloudHub access] is enabled on Anypoint Platform.
+. If you do not already have access to the Anypoint-CLI command line tool, install it using xref:runtime-manager::anypoint-platform-cli.adoc#installation[Anypoint Platform CLI installation instructions].
```

Comment:
This topic is in the runtime-manager repo so you don't need runtime-manager in the xref. Also, we should probably point them to xref:anypoint-platform-cli2.adoc

Reply:
fixed and tested locally